packages/gallery/src/components/item/itemView.js

```diff
@@ -982,6 +983,26 @@ class ItemView extends GalleryComponent {
     return tabIndex;
   }
 
+  onAnchorFocus() {
+    if (
+      (isSiteMode() || isSEOMode()) &&
+      !utils.isMobile() &&
+      window.document &&
```

Comment:
This is a duplicated code. Please reuse the condition written in `changeActiveElementIfNeeded`